orchagent/dash/dashaclgroupmgr.cpp

```diff
@@ -565,6 +601,32 @@ task_process_status DashAclGroupMgr::removeRule(const string& group_id, const st
     return task_success;
 }
 
+bool DashAclGroupMgr::fetchRule(const std::string &group_id, const std::string &rule_id, DashAclRule &rule)
+{
+    auto key = group_id + ":" + rule_id;
+    vector<FieldValueTuple> tuples;
+
+    bool exists = m_dash_acl_rules_table->get(key, tuples);
```

Comment:
The rules that we have in the cache already have been validated. If the rule doesn't exist or can't be parsed then something bad happened with the DB. We should crash in this situation. We can't recover from this state

Reply:
We will crash since the status is propagated to the functions that return "task_failed". In this case there also will be more error messages before the crash to help understand what happened.